pygmt/tests/test_clib_put_matrix.py

```diff
@@ -55,7 +56,7 @@ def test_put_matrix_fails():
 
 
 def test_put_matrix_grid():
-    "Check that assigning a numpy 2d array to a grid works"
+    "Check that assigning a numpy 2d array to an ASCII and NetCDF grid works"
     dtypes = "float32 float64 int32 int64 uint32 uint64".split()
```

Comment:
Should probably use [pytest parametrize](https://docs.pytest.org/en/stable/parametrize.html) for this, but we can leave it to a separate PR. Our test suite is getting larger and we might want to consider more efficient ways of testing (I'll open an issue for it).